File: android/app/src/main/java/com/dtools/DeviceOwner/DeviceOwnerReceiver.java
package com.dtools.DeviceOwner;

import android.app.admin.*;
import android.content.*;

public class DeviceOwnerReceiver extends DeviceAdminReceiver {

    @Override
    public void onProfileProvisioningComplete(Context context, Intent intent) {
        DevicePolicyManager manager = (DevicePolicyManager) context.getSystemService(Context.DEVICE_POLICY_SERVICE);
        ComponentName componentName = getComponentName(context);
        manager.setProfileName(componentName, "DTools");
    }

    public static ComponentName getComponentName(Context context) {
        return new ComponentName(context.getApplicationContext(), DeviceOwnerReceiver.class);
    }

}

File: android/app/src/main/kotlin/com/dtools/DeviceOwner/DeviceOwnerReceiver.kt
package com.dtools.DeviceOwner

import android.app.admin.*
import android.content.*

class DeviceOwnerReceiver : DeviceAdminReceiver() {

    override fun onProfileProvisioningComplete(context: Context, intent: Intent) {
        val manager = context.getSystemService(Context.DEVICE_POLICY_SERVICE) as DevicePolicyManager
        val componentName = getComponentName(context)
        manager.setProfileName(componentName, "DTools")
    }

    companion object {

        fun getComponentName(context: Context): ComponentName {
            return ComponentName(context.applicationContext, DeviceOwnerReceiver::class.java)
        }
    }

}